pkg/api/v1/schema.go

```diff
@@ -81,6 +83,33 @@ type DiskSourceHost struct {
 
 // END Disk -----------------------------
 
+// BEGIN Serial -----------------------------
+
+type Serial struct {
+	Type         string       `json:"type"`
+	SerialTarget SerialTarget `json:"serialTarget"`
+}
+
+type SerialTarget struct {
+	Port string `json:"port"`
+}
+
+// END Serial -----------------------------
+
+// BEGIN Console -----------------------------
+
+type Console struct {
+	Type          string        `json:"type"`
+	ConsoleTarget ConsoleTarget `json:"consoleTarget"`
```

Comment:
Naming int `Target` in the struct and `target` in the json tag should be good enough.